erpnext/assets/doctype/location/utils_location.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+@frappe.whitelist()
+def get_total_location():
```

Comment:
This would show wrong result if location filter is applied.
Also no need for new file, we can add this method in `location.py` itself.

Reply:
Suggested changes in: e72bd3c